### ISSUE TITLE
enhancement request: concurrent jobs setting

### ISSUE THREAD

**Issue by krixoko** [WynterJones/Midjourney-AutoPilot#2]
It would be nice to set the concurrent number of jobs/worker, since it is possible to run 3 concurrent jobs.

**Comment by WynterJones** [comment]
You are correct! I didn't even think of it that -- the way the code is written it does run one job and splits message list by a timestamp, so to support concurrent I would need to re-think that approach. 

I'll have a good think about it! Thanks for the great suggestion
